src/test/util/validation.cpp

```diff
@@ -21,6 +21,7 @@ void TestChainstateManager::DisableNextWrite()
 void TestChainstateManager::ResetIbd()
 {
     m_cached_finished_ibd = false;
+    m_cached_chaintip_recent = false;
```

Comment:
I find the existence of his whole method very hacky, we're testing something that cannot happen in reality so if the test passes or fails after this, it won't increase my confidence in the product.
But if you insist on updating it, we should likely update `JumpOutOfIbd` as well for symmetry.